src/pages/game/point-defender-info.jsx

```diff
@@ -1,36 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import random from 'lodash.random';
 
 export function PointDefenderInfo({ info }) {
+  const [deciderName] = useState(getRandomName());
```

Comment:
is this an OK use of `useState` ?

Reply:
If you only want it to set once when the component mounts? Sure!